src/libraries/Common/src/Interop/Windows/Advapi32/Interop.GetSecurityInfoByHandle.cs

```diff
@@ -9,8 +9,7 @@ internal static partial class Interop
 {
     internal static partial class Advapi32
     {
-        [DllImport(Interop.Libraries.Advapi32, EntryPoint = "GetSecurityInfo", CallingConvention = CallingConvention.Winapi,
-            SetLastError = true, ExactSpelling = true, CharSet = CharSet.Unicode)]
+        [DllImport(Interop.Libraries.Advapi32, EntryPoint = "GetSecurityInfo", SetLastError = true, ExactSpelling = true, CharSet = CharSet.Unicode)]
```

Comment:
`CallingConvention.Winapi` is the default value, no need to specify it.

Reply:
`CharSet` isn't needed either as there are no `char` or `string` arguments.